### ISSUE TITLE
Treemacs extension trigger an error when selecting treemacs window

### ISSUE THREAD

**Issue by Kaycell** [doomemacs/themes#699]
### What did you expect to happen?

When I do `M-x  treemacs-select-window`, it should switch to treemacs window silently

### What actually happened?

When I do `M-x  treemacs-select-window`, it switch to treemacs window but the following error message appear:

```
apply: Wrong number of arguments: ((t) nil "Remove fringes in current window if `treemacs-fringe-indicator-mode' is nil" (if (display-graphic-p) (progn (if treemacs-fringe-indicator-mode (set-window-fringes nil doom-themes-treemacs-bitmap-indicator-width 0) (set-window-fringes nil 0 0))))), 1
```
![Screenshot 2022-01-05 at 10 44 38](https://user-images.githubusercontent.com/8752003/148196534-d03daa99-313a-406e-ad0e-10c7a6bc1517.png)

### Describe your attempts to resolve the issue

I tried with and without calling  `(doom-themes-treemacs-config)` in my init.el and the issue is located in https://github.com/doomemacs/themes/blob/8644ac46de4318e92501e3e577cde74738ab8622/extensions/doom-themes-ext-treemacs.el#L69
if I'm correct.
For whatever reason this hook seems to be called with an argument. I also think it's the reason treemacs fringe indicator don't work properly with doom-theme.

### Steps to reproduce

1. Use emacs 27.2
2. Install doom-theme:
```
  (use-package doom-themes
    :config
    (load-theme 'doom-one t)
    (doom-themes-visual-bell-config)
    (setq doom-themes-treemacs-theme "doom-colors")
    (doom-themes-treemacs-config))
```
3. Install treemacs:
```
(use-package treemacs
    :defer t
    :init
    (with-eval-after-load 'winum
      (define-key winum-keymap (kbd "M-0") #'treemacs-select-window))
    :config
    (progn
      (setq treemacs-is-never-other-window t
            treemacs-user-mode-line-format 'none)
      (treemacs-fringe-indicator-mode 'always))
    :bind
    (:map global-map
          ("M-0"       . treemacs-select-window)
          ("C-x t 1"   . treemacs-delete-other-windows)
          ("C-x t t"   . treemacs)
          ("C-x t B"   . treemacs-bookmark)
          ("C-x t C-t" . treemacs-find-file)
          ("C-x t M-t" . treemacs-find-tag)))
```
4. In emacs `M-x  treemacs-select-window`

### Package commit

8644ac46de4318e92501e3e577cde74738ab8622

### System Information

I use OSX 11.6.1 and use this project when my dog is not around

**Comment by hlissner** [comment]
Please generate a backtrace of the error:

1. `M-x toggle-debug-on-error`
2. Reproduce the error message.

**Comment by hlissner** [comment]
Looks like `treemacs-select-window` (which we advise) was changed to take an argument in https://github.com/Alexander-Miller/treemacs/commit/beaed17f67106caa65c27ac45388a0705b36d369. I've fixed this in fdc0739, which should resolve your issue. Let me know if that isn't the case and I'll reopen it. Thanks for bringing it to my attention!
